pkg/util/ssh.go

```diff
@@ -247,3 +267,91 @@ func GenerateKey(bits int) (*rsa.PrivateKey, *rsa.PublicKey, error) {
 	}
 	return private, &private.PublicKey, nil
 }
+
+type TestSSHServer struct {
```

Comment:
Can this struct and the function below go into ssh_test.go? Is there a pkg/conditional compilation issue?